backend/src/main/java/com/woowacourse/kkogkkog/member/application/MemberService.java

```diff
@@ -110,6 +112,15 @@ public List<MemberResponse> findAll() {
             .collect(toList());
     }
 
+    @Transactional(readOnly = true)
+    public MembersResponse findByNickname(String searchName) {
+        List<Member> members = memberRepository.findByNickname(searchName);
+
+        return new MembersResponse(members.stream()
+            .map(MemberResponse::of)
+            .collect(Collectors.toList()));
+    }
```

Comment:
MembersResponse는 presentation 계층의 DTO이니, List<MemberResponse>로 수정하죠.

```suggestion
    @Transactional(readOnly = true)
    public List<MemberResponse> findByNickname(String searchName) {
        List<Member> members = memberRepository.findByNickname(searchName);

        return members.stream()
            .map(MemberResponse::of)
            .collect(Collectors.toList());
    }
```

Reply:
반영했습니다👍